tools/importers/SynchronizeColorsTokens.ts

```diff
@@ -301,4 +247,4 @@ const SynchronizeColorsTokens = async () => {
 	});
 };
 
-SynchronizeColorsTokens();
+SynchronizeColorsTokens().then(() => console.log('Import in progress...'));
```

Comment:
something isn't right here. We wait until execution is done in this method, but we write "in progress"

Reply:
I can left empty body here, but because we have async requests it always is displaying first, so in the practice we have to lines in output:
```
Import in progress...
The import was successful for bin: 633d96230e6a79321e1dec41
```

Reply:
you can `await` inside the method to mitigate this problem. Also, if you decide to not change that, we shouldn't be returning Promise - we're not waiting for execution to be finished, so it's misleading

Reply:
I fixed it, now we have information in correct order and time:
```
Import in progress...
The import was successful for bin: 633d96230e6a79321e1dec41
The importer has completed the work.
```

Reply:
it's good that order of logs is correct, but I still believe that if we return a `Promise` it should be possible to await until code in `Promise` is executed. Right now, you attach to `then()` which is fired when `Promise` is resolved, but you console log is printed before code inside this method finishes

Reply:
Wait, I made change in the tokens importer, not here. I fixed it and now it looks like this:
```
SynchronizeColorsTokens().finally(() => {
	console.log('The importer has completed the work.');
});
```

Reply:
Ok, I think console log there is redundant in this case. I removed it. Now we have info about:
```
Import in progress...
The import was successful for bin: 633d96230e6a79321e1dec41
```

Reply:
IMO, the root cause of the problem is here: https://github.com/bethinkpl/design-system/blob/IT-4898-tt-tokens-to-scss/tools/importers/SynchronizeColorsTokens.ts#L254

You define function as `async`, but inside the function you don't `await` for asynchronous functions execution. Because of that code in `then/finally` is executed before asynchronous code (`axios.get()`) is finished

Reply:
Right, but if function is not `async`  the problem is only moved on -> we are doing nothing with async `SynchronizeSingleBin()` and IDE has the same problem. Generally we need somewhere catch the promise.